source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx

```diff
@@ -236,7 +236,7 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 					telemetryBrowser.capture("EDITOR cloned repository", {
 						owner: args.routeParams.owner,
 						repository: args.routeParams.repository,
-						type: response.data.private ? "isPrivate" : "isPublic",
+						type: response.data.private ? "Private" : "Public",
```

Comment:
minor: I would make the strings lowercase because "unknown" is lowercase. Really minor though